frontend/src/features/leases/detail/LeasePages/insurance/edit/InsuranceForm.tsx

```diff
@@ -59,10 +59,9 @@ const InsuranceForm: React.FunctionComponent<IInsuranceRowProps> = ({ nameSpace
           <div>
             <strong>Limit ($)</strong>
           </div>
-          <InlineInput
-            type="number"
+          <FastCurrencyInput
+            formikProps={formikProps}
             field={withNameSpace(nameSpace, 'coverageLimit')}
-            data-testid="insurance-form-limit"
```

Comment:
Without this line the the test will fail
``` const formLimits = result.getAllByTestId('insurance-form-limit');```

Reply:
test failure corrected.
Can't use testid as it is not supported by the masked input.